### PR TITLE
Small improves to ideas for Julia

### PR DESCRIPTION
**Pull request by rgaiacs**
@mlubin and @madeleineudell Could you review this?

### CODE REVIEW

**Comment by madeleineudell** [comment]
looks good to me!

On Thu, Feb 11, 2016 at 5:45 AM, Raniere Silva <notifications@github.com>
wrote:

> @mlubin <https://github.com/mlubin> and @madeleineudell
> <https://github.com/madeleineudell> Could you review this?
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/numfocus/gsoc/pull/68
> Commit Summary
>
>    - Improve ideas for Julia
>
> File Changes
>
>    - *M* 2016/ideas-list-juliaopt.md
>    <https://github.com/numfocus/gsoc/pull/68/files#diff-0> (68)
>
> Patch Links:
>
>    - https://github.com/numfocus/gsoc/pull/68.patch
>    - https://github.com/numfocus/gsoc/pull/68.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numfocus/gsoc/pull/68>.
>



-- 
Madeleine Udell
Postdoctoral Fellow at the Center for the Mathematics of Information
California Institute of Technology
*https://courses2.cit.cornell.edu/mru8
<https://courses2.cit.cornell.edu/mru8>*
(415) 729-4115

File: 2016/ideas-list-juliaopt.md [hunk 1]
```diff
@@ -16,9 +17,29 @@ and in particular from the modeling interfaces
...
+</tr>
+
+<tr>
+    <td>Involves</td>
+    <td>Julia, C, SCIP, Linear and Integer Programming, Combinatorial Optimization</td>
+</tr>
+
+<tr>
+    <td>Mentors</td>
+    <td>[@mlubin], [@joehuchette], [@rgaiacs]</td>
```
Comment (mlubin) [inline 2016/ideas-list-juliaopt.md]: Are these meant to be links?

Comment (rgaiacs) [inline 2016/ideas-list-juliaopt.md]: Yes. I can use standard Markdown. But I was hoping that GItHub understand this.

Comment (mlubin) [inline 2016/ideas-list-juliaopt.md]: They don't appear as links in the preview

Comment (pbanaszkiewicz) [inline 2016/ideas-list-juliaopt.md]: See https://github.com/numfocus/gsoc/pull/67#discussion_r52642400

**Pull request merged**